lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -193,7 +194,7 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         /// Return current Job status
         /// </summary>
         /// <returns></returns>
-        /// TODO: REEF-889
+        /// TODO: [REEF-889]
```

Comment:
@markusweimer Do you mean add a layer of interface  IREEFClientYARN  : IREEFClient and put YARN specific APIs?